src/cloud_provider/helm.rs

```diff
@@ -332,8 +332,35 @@ pub fn deploy_charts_levels(
     envs: &Vec<(String, String)>,
     charts: Vec<Vec<Box<dyn HelmChart>>>,
     dry_run: bool,
+    max_nodes: Option<usize>,
 ) -> Result<(), SimpleError> {
-    // first show diff
+    // first, check if enough nodes are available
+    let kubectl_envs = envs
+        .into_iter()
+        .map(|env| (env.clone().0.as_str(), env.clone().1.as_str()))
+        .collect();
+
+    if max_nodes.is_some() {
+        match kubectl_exec_get_node(&kubeconfig, kubectl_envs) {
+            Err(e) => {
+                error!("Can't get nodes for helm deployments");
+                return Err(e);
+            }
+            Ok(nodes) => match nodes.items.len() < max_nodes.unwrap() {
+                true => Ok(()),
+                false => {
+                    error!(format!(
+                        "Not enough nodes to perform Helm deployments. {}/{} used.",
```

Comment:
Can you please rephrase to make it more clear that it's a security that Qovery add in advance to avoid issues plz